src/RockPaperScissors/RpsServer/Controllers/RpsController.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using RpsServer.Models;
+
+namespace RpsServer.Controllers
+{
+    [Route("api/[controller]")]
+    public class RpsController : Controller
+    {
+        private readonly GameContext context;
+
+        public RpsController(GameContext context)
+        {
+            this.context = context;
+        }
+
+        #region APIs
+        [HttpGet]
+        public IActionResult CreateUser()
+        {
+            User newUser = new User();
+            this.context.Users.Add(newUser);
+            this.context.SaveChanges();
+            return new ObjectResult(newUser.Id);
+        }
+
+        [HttpGet("{playerId}")]
+        public IActionResult GetOrCreateGame(Guid playerId)
```

Comment:
This functionality could be merged with status and make move to make things even simpler.